ape_hardhat/__init__.py

```diff
@@ -24,6 +24,8 @@ def config_class():
 def providers():
     yield "ethereum", LOCAL_NETWORK_NAME, HardhatProvider
     yield "ethereum", "mainnet-fork", HardhatMainnetForkProvider
+    yield "fantom", LOCAL_NETWORK_NAME, HardhatProvider
+    yield "fantom", "opera-fork", HardhatMainnetForkProvider
```

Comment:
do we need to add `opera-fork` to `ape-fantom` for this to work?

Reply:
I already did haha